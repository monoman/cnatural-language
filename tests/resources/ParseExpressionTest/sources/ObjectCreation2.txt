new T { P = 1 }