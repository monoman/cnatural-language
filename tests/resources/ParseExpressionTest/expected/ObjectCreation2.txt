new T{P=Literal(DecimalInteger, [1])}